spec/services/initials_avatar_generator_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+describe InitialsAvatarGenerator do
+  it 'creates proper color' do
+    expect(ReceiveAllowedColors.call).to include(described_class.call)
```

Comment:
Ten test to "zabetonowanie" implementacji. Powinniśmy testować zachowania:

`expect(described_class.new.call).to be_in(described_class::POSSIBLE_COLORS)`